get_ci_vm/README.md

```diff
@@ -1,10 +1,151 @@
+         executing from **within** a newly created VM with CWD undefined.
+      1. Execute any commands necessary (on the VM) to prepare it for
+         use by the user.  For example, this could include calling *make* targets
+         and/or environment-setup scripts like `./contrib/cirrus/setup.sh`.
+      1. Exit non-zero if there is an error.
+   1. When called with the `--config` option, the script *must*:
+      1. Confirm `$GET_CI_VM` is `1`.  This signifies the script is (now)
+         executing from **within** the get_ci_vm container context.
+      1. Print on *stdout*, a list of `key=value` env. var lines conforming
+         to [the *APIV1 Env. Vars.* section below](README.md#env-vars).
+      1. Exit non-zero if there is an error.
+
+### Env. Vars.
+
+When `hack/get_ci_vm.sh` is executed with the `--config` argument, it
+*must* emit a `key=value` list of env. vars. to *stdout* if and only if
+the `$GET_CI_VM` value is set to `1`.
+
+* `DESTDIR` - *Must* be set to the "clone" directory (on the VM) for the
+  repository's code.  This directory is analogous to the `$CIRRUS_WORKING_DIR`
+  value in `.cirrus.yml`.  It will created if it doesn't exist, and
+  a copy of the user's repository will be presented, under a `get_ci_vm`
+  branch.
+* `UPSTREAM_REPO` - *Must* be set to the upstream clone URL for the repository.
+  This will be set as a remote and fetched (on the VM) to facilitate use of the
+  VM.  It's also used to prevent uploading large amounts of data from the user's
+  local system.
+* `CI_ENVFILE` - This *may* point to a file containing any CI-runtime env. vars.
+  which should also be present in the (eventual) remote shell for the user.
+  For example, the podman repository's `./contrib/cirrus/setup_environment.sh`
+  populates a `/etc/ci_environment` file with important CI setup-time values.
+* `GCLOUD_PROJECT` - *Must* contain the name of the GCE project space where
+  the VM will be created.
+* `GCLOUD_IMGPROJECT` - *Must* be set to `libpod-218412` (the GCE project space
+  containing all the VM images).
+* `GCLOUD_CFG` - *Must* be set to the name of the GCE configuration to maintain
+  for this repository.  See the GCE settings bind-mount item under
+  the *Operational Details* section above.
+* GCLOUD_ZONE - *Must* be set to either `us-central1-a` or any valid network-near
+  zone to the user.   Valid values are listed in [the
+  documentation](https://cloud.google.com/compute/docs/regions-zones/#available)
+* `GCLOUD_CPUS` - *Must* be set to the number of vCPUs to assign to the VM.
+* `GCLOUD_MEMORY` - *Must* be set to the amount of memory to assign to the VM.
+* `GCLOUD_DISK` - *Must* be set to `200` (gigabytes).
+
+## APIv2 (AWS EC2 Support)
+
+***N/B:*** The repository's `.cirrus.yml` **must** define a
+**`EC2_INST_TYPE`** env. var. for any EC2-based tasks or globally.
+
+### Hack-script requirements
+
+1. Somewhere within the script the string `get_ci_vm APIv2` *must* be present.
```

Comment:
If I'm reading `entrypoint.sh` correctly, the actual requirement is `# get_ci_vm APIv2` (hash / comment). If so, please adjust this text (and line 87 above).

Reply:
correct, thanks I'll fix it.